src/main/java/core/basesyntax/report/ReportCreator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.report;
+
+import java.util.Map;
+
+public class ReportCreator {
+    public String createReport(Map<String, Integer> storage) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("fruit,quantity\n");
+        storage.forEach((key, value) -> {
+            sb.append(key).append(",").append(value).append("\n");
+        });
+        return sb.toString();
```

Comment:
sb bad naming
extract literals to constants REPORT_HEADER etc
use System.lineSeparator()
you can append right after constructor fyi